src/components/Accounts/ChargeFeeFine/ChargeFeeFine.js

```diff
@@ -132,7 +132,8 @@ class ChargeFeeFine extends React.Component {
     type.location = item?.location?.name || item?.effectiveLocation?.name;
     type.materialType = (item.materialType || {}).name;
     type.materialTypeId = (selectedLoan.id) ? undefined : (item.materialType || {}).id || undefined;
-
+    
```

Comment:
Please fix these lint warnings. It should be possible to configure your editor to trim trailing whitespace.

Reply:
Yeah, sorry. My editor also adds trailing comma, so I just turned linter off when make last corrects in this file. Ofcourse this warnings will be fixed. Thanks.